kubernetes/samples/charts/traefik/README.md

```diff
@@ -91,6 +91,84 @@ TLS_PORT=`kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.po
 $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:${TLS_PORT}/testwebapp/
 ```
 
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at the ingress controller to access the WebLogic Server Administration Console through the SSL port. 
+
+### 1. Enable "WebLogic Plugin Enabled" on the WebLogic domain level
+
+If you are using WDT to configure the WebLogic domain, you need to add the following resource section at the domain level to the model YAML file.
+```
+resources:
+     WebAppContainer:
+         WeblogicPluginEnabled: true
+```
+If you are using a WLST script to configure the domain, then the following modifications are needed to the respective PY script.
+```
+# Configure the Administration Server
+cd('/Servers/AdminServer')
+set('WeblogicPluginEnabled',true)
+...
+cd('/Clusters/%s' % cluster_name)
+set('WeblogicPluginEnabled',true)
+```
+### 2. Update the ingress resource with customRequestHeaders value
+Replace the string `weblogic-domain` with namespace of the WebLogic domain, the string `domain1` with domain UID and the string `adminserver` with name of the Administration Server in the WebLogic domain.  
+
+
+**NOTE**: If you also have HTTP requests coming into an ingress, make sure that you remove any incoming `WL-Proxy-SSL` header. This protects you from a malicious user sending in a request to appear to WebLogic as secure when it isn't. Add the following `customRequestHeaders` in the Traefik ingress configuration to block `WL-Proxy` headers coming from the client. In the following example, the ingress resource will eliminate the client header `WL-Proxy-Client-IP`.
+
+```
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: traefik
+  name: traefik-console-tls
+  namespace: weblogic-domain
+spec:
+  entryPoints: 
+   - websecure
+  routes:
+  - kind: Rule
+    match: PathPrefix(`/console`)
+    middlewares:
+    - name: tls-console-middleware
+      namespace: weblogic-domain
+    services:
+    - kind: Service
+      name: domain1-adminserver
+      namespace: weblogic-domain
+      port: 7001
+  tls:
+     secretName: domain1-tls-cert
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: tls-console-middleware
+  namespace: weblogic-domain
+spec:
+  headers:
+    customRequestHeaders:
+      WL-Proxy-SSL: "true"
+      X-Custom-Request-Header: ""
+      X-Forwarded-For: "" 
+      WL-Proxy-Client-IP: ""
```

Comment:
Is it possible to filter out WL-Proxy-SSL and WL-Proxy-Client-IP?

Reply:
done